pm/queue.go

```diff
@@ -124,13 +124,16 @@ func (q *ticketQueue) handleBlockEvent(latestL1Block *big.Int) {
 	for i := 0; i < int(numTickets); i++ {
 		nextTicket, err := q.store.SelectEarliestWinningTicket(q.sender, new(big.Int).Sub(q.tm.LastInitializedRound(), big.NewInt(ticketValidityPeriod)).Int64())
 		if err != nil {
-			glog.Errorf("Unable select earliest winning ticket err=%q", err)
+			glog.Errorf("Unable to select earliest winning ticket err=%q", err)
 			return
 		}
 		if nextTicket == nil {
 			return
 		}
-
+		if !q.isRecipientActive(nextTicket.Recipient) {
+			glog.V(5).Infof("Ticket recipient is not active in this round, cannot redeem ticket recipientRandHash=%x senderNonce=%v", nextTicket.RecipientRandHash, nextTicket.SenderNonce)
```

Comment:
```suggestion
			glog.V(5).Infof("Ticket recipient is not active in this round, cannot redeem ticket recipient=%v", nextTicket.Recipient.Hex())
```

seems like the most useful information here is the recipient as opposed to the other fields.